src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,169 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capacity;
+    private int threshold;
```

Comment:
remove these fields, they can be obtained from other fields

Reply:
I don't understand what you mean...